src/gleam/float.gleam

```diff
@@ -401,3 +401,22 @@ if javascript {
   external fn do_random_uniform() -> Float =
     "../gleam_stdlib.mjs" "random_uniform"
 }
+
+/// Returns division of the inputs as a `Result`.
+///
+/// ## Examples
+///
+/// ```gleam
+/// > divide(0.0, 1.0)
+/// Ok(1.0)
+///
+/// > divide(1.0, 0.0)
+/// Error(Nil)
+/// ```
+///
+pub fn divide(number a: Float, by b: Float) -> Result(Float, Nil) {
```

Comment:
Could you remove the noun label `number` please? Thanks

Reply:
Sorry overlooked, removed!